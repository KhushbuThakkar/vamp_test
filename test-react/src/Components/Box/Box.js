import React from "react";
import PropTypes from "prop-types";
import { Box as RBox } from "rebass";

const Box = (props) => {
  return <RBox {...props}>{props.children}</RBox>;
};

Box.propTypes = {
  children: PropTypes.element.isRequired,
};

export default Box;
