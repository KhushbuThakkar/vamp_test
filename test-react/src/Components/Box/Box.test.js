import React from "react";
import renderer from "react-test-renderer";
import Box from "./Box";

it("render correctly Box component", () => {
  const component = renderer.create(
    <Box px={2} py={2} width={1 / 2}>
      <div>hello</div>
    </Box>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
