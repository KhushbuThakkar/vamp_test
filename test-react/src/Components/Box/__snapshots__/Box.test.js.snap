// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Box component 1`] = `
<div
  className="css-1axab2n"
>
  <div>
    hello
  </div>
</div>
`;
